.github/workflows/release.yml

```diff
@@ -0,0 +1,81 @@
+---
+name: "TER deployment"
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  set-matrix:
+    runs-on: ubuntu-20.04
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
```

Comment:
Should the output really be a matrix, or should/can it be an env var instead?

Reply:
As I commented above